simple_salesforce/login.py

```diff
@@ -12,7 +12,7 @@
 import requests
 
 
-def SalesforceLogin(**kwargs):
+def SalesforceLogin(session=None, **kwargs):
```

Comment:
Based on the convention below I've been doing `kwargs.get('session')` instead of defining it inside the actual definition line.

I'm open to switching obviously. But mixing both doesn't make a ton of sense.

Also you'll probably want to add something to the docstring about this.